nextcord/message.py

```diff
@@ -1850,13 +1857,14 @@ async def edit(self, **fields: Any) -> Optional[Message]:
             if content is not None:
                 fields['content'] = str(content)
 
-        try:
-            embed = fields['embed']
-        except KeyError:
-            pass
-        else:
-            if embed is not None:
-                fields['embed'] = embed.to_dict()
+        if 'embed' in fields and 'embeds' in fields:
+            raise InvalidArgument('Cannot pass both embed and embeds parameter to edit()')
+
+        if 'embed' in fields and fields['embed'] is not None:
+            fields['embed'] = fields['embed'].to_dict()
```

Comment:
```suggestion
            fields['embed'] = [fields['embed'].to_dict()]
```
api v10 thing

Reply:
The send_multipart_helper converts embed to embeds already so it doesn't need to be handled here

Reply:
k

Reply:
Actually, it seems like it only gets converted if files are sent. I'll look into this further  :+1: